groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -213,13 +294,45 @@ private static SymbolKind getKind(ClassNode node) {
         return SymbolKind.Class;
     }
 
-    private Set<SymbolInformation> getMethodSymbolInformations(String sourcePath, ClassNode parent, MethodNode method) {
+    private SymbolInformation getVariableSymbolInformation(String parentName, String sourcePath, Variable variable) {
+        Location location;
+        SymbolKind kind;
+        if (variable instanceof DynamicVariable) {
+            kind = SymbolKind.Field;
```

Comment:
consider bypassing using `createSymbolInformation` here and just using the builder directly. e.g. 
```
SymbolInformationBuilder builder = new SymbolInformationBuilder().containerName(parentName);
if (variable instanceof DynamicVariable) {
    builder.kind(SymbolKind.Field);
    builder.location(...);
}
```